sidechain/consensus/aura/src/lib.rs

```diff
@@ -48,17 +48,16 @@ use sp_runtime::{
 use std::{string::ToString, sync::Arc, time::Duration, vec::Vec};
 
 pub mod block_importer;
+pub mod block_production_suspension;
+pub mod peer_block_sync;
 pub mod proposer_factory;
 pub mod slot_proposer;
 mod verifier;
 
 pub use verifier::*;
 
 #[cfg(test)]
-mod mock;
-
-#[cfg(test)]
-mod block_importer_tests;
+mod test;
```

Comment:
I re-arranged the test code in this crate, put everything in the `test` module, which in turn contains `fixtures` and `mocks`.